docs/getting-started.md

```diff
@@ -12,13 +12,22 @@ menuItems:
 
 # Getting Started with Serverless
 
-First things first, lets get the serverless framework installed on your machine.
+First things first, let's get the Serverless framework installed on your machine.
 
 ```bash
 # Installing the serverless cli
 npm install serverless -g
 ```
 
+## Installing Serverless in an existing project
```

Comment:
I think "Invoking Serverless locally from a project" should be the next adjacent section:

## Invoking Serverless locally from within a project

To execute the locally installed Serverless executable you have to reference the binary out of the node modules directory.

Example:
```
node ./node_modules/serverless/bin/serverless deploy
```